src/Dictionaries/IDictionary.h

```diff
@@ -252,6 +252,49 @@ class IDictionary : public IExternalLoadable, public IKeyValueEntity
         return result;
     }
 
+    /**
+      * Analogous to getColumn, but for dictGetOrDefault's short circuit case
+      */
```

Comment:
Please, add a comment about `default_mask` argument, what is it and how it should be filled